src/agenda.ts

```diff
@@ -8,25 +8,29 @@ export class Contact {
         this.surname=theSurname;
         this.address=theAddress;
     }
-    
-    // public displayContact(surname){
-    //     this.surname=surname;
-    //     return this.name + ' ' + this.surname + ' ' + this.address;
-    // }
 
 }
 
 
 export class Agenda {
     contacts: Contact[] = [];
     
-    public addContact(newName: string, newSurname: string, newAddress: string){
-        this.contacts.push(new Contact(newName, newSurname, newAddress));
+    public addContact(Contact){
```

Comment:
contact is not a string, is a Contact

the syntax is like this:

`variableName: ClassName`

In the case of `string`, it is lowercase for other reason we may learn later, so ignore that case.